sources/packages/backend/apps/api/src/route-controllers/education-program-offering/_tests_/e2e/education-program-offering.institutions.validateOffering.e2e-spec.ts

```diff
@@ -0,0 +1,125 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
```

Comment:
If we are testing an API end point, the e2e test file name should be having *controller*